subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -65,6 +66,8 @@ static uint32_t poll_sleep = (CONFIG_HAWKBIT_POLL_INTERVAL * 60 * MSEC_PER_SEC);
 static uint32_t poll_sleep = (300 * MSEC_PER_SEC);
 #endif
 
+static char hawkbit_server[DNS_MAX_NAME_SIZE + 1];
+static char hawkbit_server_port[6];
```

Comment:
I'm not sure if there's a definition for the max value of a port:
```suggestion
static char hawkbit_server_port[sizeof(STRINGIFY(__UINT16_MAX__))];
```